### PR TITLE
Remove underscore & fix exports

### PR DESCRIPTION
**Pull request by StorytellerCZ**
Removes the need for underscore and fixes an import/export issue.

### CODE REVIEW

File: server/user-presence.js [hunk 1]
```diff
@@ -155,3 +153,6 @@ ServerPresence.onCleanup((serverId) => {
         UserSessions.remove({});
     }
 });
+
+/* eslint-disable import/prefer-default-export */
+export default UserPresence;
```
Comment (copleykj) [inline server/user-presence.js]: I'm not sure what issue this fixes here. The export seemed to work fine on my end.

File: package.js [hunk 2]
```diff
@@ -12,5 +12,5 @@ Package.onUse(function _(api) {
     api.use(['socialize:server-presence@1.0.0', 'socialize:user-model@1.0.0']);
 
     api.mainModule('server/server.js', 'server');
-    api.mainModule('common/common.js');
+    api.mainModule('common/common.js', 'client');
```
Comment (copleykj) [inline package.js]: After looking at the code structure, it looks like anything that is common code is actually already imported into server so while the current code works, it's slightly redundant.  We can rename common to client to reflect this. Nice catch here.

File: server/user-presence.js (hunk 1 above)
Comment (StorytellerCZ) [reply server/user-presence.js]: It was an issue on my side in the past. Will have to check if it still is with the newer versions.

**Comment by StorytellerCZ** [comment]
Fixed!

File: server/server.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import { Meteor } from 'meteor/meteor';
 
 /* eslint-enable import/no-unresolved */
 
-import { determineStatus, UserPresence } from './user-presence.js';
+import UserPresence, { determineStatus } from './user-presence.js';
```
Comment (copleykj) [inline server/server.js]: UserPresence would be undefined in this case since `user-presence.js` exports it as a named export

Comment (StorytellerCZ) [reply server/server.js]: Darn! Missed that! Fixed!

**Comment by copleykj** [comment]
I'm gonna close this since the final result is just switching the position of named exports.

**Comment by StorytellerCZ** [comment]
Fair enough.

**Pull request closed without merging**